inc/Engine/Media/AboveTheFold/Activation/ServiceProvider.php

```diff
@@ -0,0 +1,59 @@
+<?php
+declare(strict_types=1);
+
+namespace WP_Rocket\Engine\Media\AboveTheFold\Activation;
+
+use WP_Rocket\Dependencies\League\Container\ServiceProvider\AbstractServiceProvider;
+use WP_Rocket\Engine\Media\AboveTheFold\Database\Queries\AboveTheFold as ATFQuery;
+use WP_Rocket\Engine\Media\AboveTheFold\Context\Context;
+use WP_Rocket\Engine\Media\AboveTheFold\WarmUp\Controller as WarmUpController;
+use WP_Rocket\Engine\Media\AboveTheFold\Jobs\Manager;
+
+class ServiceProvider extends AbstractServiceProvider {
+
+	/**
+	 * The provides array is a way to let the container
+	 * know that a service is provided by this service
+	 * provider. Every service that is registered via
+	 * this service provider must have an alias added
+	 * to this array or it will be ignored.
+	 *
+	 * @var array
+	 */
+	protected $provides = [
+		'atf_query',
+		'atf_context',
+		'warmup_controller',
+		'atf_manager',
```

Comment:
`atf_manager` doesn't exist anymore as part of https://github.com/wp-media/wp-rocket/issues/6461